presto-main/src/test/java/com/facebook/presto/execution/TestQueryQueueRule.java

```diff
@@ -30,27 +28,15 @@
     @Test
     public void testBasic()
     {
-        Session session = new Session("bob", Optional.of("the-internet"), "", "", UTC_KEY, ENGLISH, Optional.empty(), Optional.empty(), 0, ImmutableMap.of(), ImmutableMap.of());
         QueryQueueDefinition definition = new QueryQueueDefinition("user.${USER}", 1, 1);
         QueryQueueRule rule = new QueryQueueRule(Pattern.compile(".+"), null, ImmutableMap.of(), ImmutableList.of(definition));
-        assertEquals(rule.match(session), ImmutableList.of(definition));
+        assertEquals(rule.match(TEST_SESSION), ImmutableList.of(definition));
     }
 
     @Test
     public void testBigQuery()
     {
-        Session session = new Session(
-                "bob",
-                Optional.of("the-internet"),
-                "",
-                "",
-                UTC_KEY,
-                ENGLISH,
-                Optional.empty(),
-                Optional.empty(),
-                0,
-                ImmutableMap.of("big_query", "true"),
-                ImmutableMap.of());
+        Session session = TEST_SESSION.withSystemProperty("big_query", "true");
```

Comment:
This has a different user, source, catalog and schema. Not sure if this matters for this test.